tests/test_any.py

```diff
@@ -0,0 +1,60 @@
+from ansys.dpf import core as dpf
+
+
+def test_create_any(server_type):

```

Comment:
These unit tests won't pass the retrocompatibility pipelines I think, right?

Reply:
@cbellot000 How so?